doc/content/gateways/lora-basics-station/_index.md

```diff
@@ -4,6 +4,20 @@ description: ""
 weight: -1
 ---
 
-The [{{% lbs %}}](https://lora-developers.semtech.com/resources/tools/basic-station/welcome-basic-station/) protocol simplifies management of large scale LoRaWAN networks. {{% lbs %}} contains two sub protocols for connecting Gateways to Network Servers, LoRaWAN Network Server (LNS) and Configuration and Update Server (CUPS).
+The [{{% lbs %}}](https://lora-developers.semtech.com/resources/tools/basic-station/welcome-basic-station/) protocol simplifies management of large scale LoRaWAN networks. {{% lbs %}} is the preferred way of connecting Gateways to {{% tts %}}. 
+
+<!--more -->
+
+Some of the advantages of {{% lbs %}} over the legacy UDP Packet Forwarder are:
+
+- Support for Class A, B, and C
```

Comment:
This is dependent on the specific gateway model and not on the protocol

Reply:
this is straight from https://lora-developers.semtech.com/resources/tools/basic-station/welcome-basic-station/

Reply:
No that's not what I meant. Yes this is a core feature of LBS, but is not an advantage over the UDP Packet forwarder.

Reply:
> No that's not what I meant. Yes this is a core feature of LBS, but is not an advantage over the UDP Packet forwarder.

I got ya. Removed it.